samples/java_springboot/44.prompt-users-for-input/README.md

```diff
@@ -26,65 +40,30 @@ This bot has been created using [Bot Framework](https://dev.botframework.com). T
-
-Replace `<botname>` and `<appsecret>` with your own values.
-
-`<botname>` is the unique name of your bot.  
-`<appsecret>` is a minimum 16 character password for your bot. 
+A bot is inherently stateless. Once your bot is deployed, it may not run in the same process or on the same machine from one turn to the next.
+However, your bot may need to track the context of a conversation, so that it can manage its behavior and remember answers to previous questions.
 
-Record the `appid` from the returned JSON
+In this example, the bot's state is used to a track number of messages.
 
-### 4. Create the Azure resources
-Replace the values for `<appid>`, `<appsecret>`, `<botname>`, and `<groupname>` in the following commands:
-
-#### To a new Resource Group
-`az deployment sub create --name "promptForInputBotDeploy" --location "westus" --template-file ".\deploymentTemplates\template-with-new-rg.json" --parameters appId="<appid>" appSecret="<appsecret>" botId="<botname>" botSku=S1 newAppServicePlanName="promptForInputBotPlan" newWebAppName="promptForInputBot" groupLocation="westus" newAppServicePlanLocation="westus"`
-
-#### To an existing Resource Group
-`az deployment group create --resource-group "<groupname>" --template-file ".\deploymentTemplates\template-with-preexisting-rg.json" --parameters appId="<appid>" appSecret="<appsecret>" botId="<botname>" newWebAppName="promptForInputBot" newAppServicePlanName="promptForInputBotPlan" appServicePlanLocation="westus" --name "promptForInputBot"`
-
-### 5. Update app id and password
-In src/main/resources/application.properties update 
-  - `MicrosoftAppPassword` with the botsecret value
-  - `MicrosoftAppId` with the appid from the first step
-
-### 6. Deploy the code
-- Execute `mvn clean package` 
-- Execute `mvn azure-webapp:deploy -Dgroupname="<groupname>" -Dbotname="<botname>"`
-
-If the deployment is successful, you will be able to test it via "Test in Web Chat" from the Azure Portal using the "Bot Channel Registration" for the bot.
-
-After the bot is deployed, you only need to execute #6 if you make changes to the bot.
+- We use the bot's turn handler and user and conversation state properties to manage the flow of the conversation and the collection of input.
+- We ask the user a series of questions; parse, validate, and normalize their answers; and then save their input.
 
+This sample is intended to be run and tested locally and is not designed to be deployed to Azure.
 
 ## Further reading
 
-- [Bot Framework Documentation](https://docs.botframework.com)
+- [Spring Boot](https://spring.io/projects/spring-boot)
+- [Maven Plugin for Azure App Service](https://github.com/microsoft/azure-maven-plugins/tree/develop/azure-webapp-maven-plugin)
 - [Bot Basics](https://docs.microsoft.com/azure/bot-service/bot-builder-basics?view=azure-bot-service-4.0)
 - [Dialogs](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-concept-dialog?view=azure-bot-service-4.0)
```

Comment:
This bullet isn't present in [C#](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/44.prompt-users-for-input/README.md#further-reading)